dist/schema.json

```diff
@@ -44,6 +45,11 @@
                     "description": "Registry provides container registry configuration."
                 }
             },
+            "required": [
+                "version",
+                "name",
```

Comment:
name should not be required since we already give the pipeline a name which is stored in the harness database.  In fact, we can probably remove name from the yaml, since the source of truth for the name is in the harness database.

Reply:
@bradrydzewski Yeah I wanted to add name as required for stage. Thanks for pointing out. Corrected it.